.circleci/config.yml

```diff
@@ -172,6 +175,10 @@ jobs:
       - run: echo $APTIBLE_PUBLIC_KEY >> ~/.ssh/known_hosts
       - run: git fetch --depth=1000000
       - run: git push git@beta.aptible.com:vita-min-demo/vita-min-demo.git $CIRCLE_SHA1:master
+      - slack/notify:
+          event: fail
+          mentions: "@badgers"
+          template: basic_fail_1
```

Comment:
Is it going to be clear what exactly failed? It would be nice to share a screenshot of what the alert might look like.

Reply:
This is using a stock message that Slack's orb provides, similar to what we see in https://cfa.slack.com/archives/G0114SW0SAU/p1692816895824119